Sources/MapboxCoreNavigation/PassiveLocationManager.swift

```diff
@@ -118,10 +128,31 @@ open class PassiveLocationManager: NSObject {
             eventsManager.sendPassiveNavigationStart()
         }
     }
-    
+
+    /**
+     Pauses the Free Drive session.
+
+     Use this method to extend the existing Free Drive session if you temporarily don't need navigation updates. For
+     more info, read the [Pricing Guide](https://docs.mapbox.com/ios/beta/navigation/guides/pricing/).
+     */
+    public func pauseTripSession() {
```

Comment:
Nit, but naming might be unclear for the end user. When I look at it, it is unclear why name contains `trip`, basically in free drive there is no trip as there is not final destination.

Reply:
Thanks for feedback! This was a [rename](https://github.com/mapbox/mapbox-navigation-ios/pull/3147#discussion_r689764134) from `pauseDriveSession` to `pauseTripSession`. This method is very billing related and you need to understand what is a trip to use it. That's why there is a link to the pricing guide (it is not up yet). And the fact that you confused by `trip` is a good thing because it will encourage you to read the docs ;)